### ISSUE TITLE
Stack trace links don't jump to source

### ISSUE THREAD

**Issue by conan** [cursive-ide/cursive#2265]
When I click on a link in a stack trace in the REPL, nothing happens.

![image](https://user-images.githubusercontent.com/3037019/68486759-aef67400-0239-11ea-9a49-4402a88aeb78.png)

For example, clicking the link in the screenshot does nothing.  It used to take me straight to the source.  Note that clicking on the Java stack trace links do work - they fire up the decompiler and jump to the decompiled source; the problem is only with Clojure.

Intellij `2019.2.4`
Cursive `1.9.0-eap8-2019.2`

**Comment by conan** [comment]
Maybe this only happens in REPLs which have failed to start?

**Comment by conan** [comment]
No, I can confirm this happens in a running REPL.  I've only noticed this with syntax errors so far.

**Comment by thheller** [comment]
Clicking any kind of file link in REPL window also has no effect anymore for me. It has been like this for a while but I cannot remember when it started. Might have never worked on Windows?

An example is doing a "Load file in REPL" that has some invalid code
```
Loading src/main/shadow/remote/relay/local.clj... 
Syntax error reading source at (C:\Users\thheller\code\shadow-cljs\src\main\shadow\remote\relay\local.clj:218:31).
Map literal must contain an even number of forms
```

The file path is properly highlighted as a link and is clickable but clicking it has no effect. Right-click also just shows "Nothing here".
